### PR TITLE
Fix Inconsistent naming for peer id and bill id

### PR DESCRIPTION
**Pull request by mubarak23**
## 📝 Description

Fix issue: #243  

---

## ✅ Checklist

Please ensure the following tasks are completed before requesting a review:

- [x] My code adheres to the coding guidelines of this project.
- [x] I have run `cargo fmt`.
- [x] I have added or updated tests (if applicable).
- [x] All CI/CD steps were successful.
- [ ] I have updated the documentation (if applicable).
- [x] I have checked that there are no console errors or warnings.
- [x] I have verified that the application builds without errors.
- [x] I've described the changes made to the API. (modification, addition, deletion).

---

## 🚀 Changes Made

- **Other Changes:**
  - change peer_id to node_id
  - change bill_name to bill_id

---

## 💡 How to Test

Please provide clear instructions on how reviewers can test your changes:

1. clone the branch
2. run `Run tests`

---

## 🤝 Related Issues

---

## 🔗 Additional Context (optional)

Add any other context or information that might be useful for reviewers:

[Additional notes or links]

---

## 📄 Relevant Documentation (optional)

Provide links to relevant documentation that reviewers may need:

- [Link to related documentation]

---

## 📋 Review Guidelines

Please focus on the following while reviewing:

- [ ] Does the code follow the repository's contribution guidelines?
- [ ] Are there any potential bugs or performance issues?
- [ ] Are there any typos or grammatical errors in the code or comments?

### CODE REVIEW

**Comment by mubarak23** [comment]
@zupzup  fixed

**Comment by zupzup** [comment]
@mubarak23 Perfect, thank you!
@mtbitcr Merged :heavy_check_mark:

File: src/service/notification_service/event.rs [hunk 1]
```diff
@@ -160,7 +160,7 @@ mod tests {
             "envelope has wrong event type"
         );
         assert_eq!(
-            &event.peer_id, &envelope.peer_id,
+            &event.node_id, &envelope.node_id,
             "envelope has wrong peer id"
```
Comment (mtbitcr) [inline src/service/notification_service/event.rs:164]: node id here

File: src/service/notification_service/event.rs [hunk 2]
```diff
@@ -175,7 +175,7 @@ mod tests {
             "deserialized event has wrong event type"
         );
         assert_eq!(
-            &deserialized_event.peer_id, &event.peer_id,
+            &deserialized_event.node_id, &event.node_id,
             "deserialized event has wrong peer id"
```
Comment (mtbitcr) [inline src/service/notification_service/event.rs:179]: node id here

**Pull request merged**